symengine/lib/symengine_wrapper.pyx

```diff
@@ -2977,11 +3029,16 @@ zoo = c2py(symengine.ComplexInf)
 nan = c2py(symengine.Nan)
 true = c2py(symengine.boolTrue)
 false = c2py(symengine.boolFalse)
+try:
+    import sympy
```

Comment:
I meant, in certain places there's code like this:
```
return exp(-I*self.position*bra.momentum/hbar)/sqrt(2*pi*hbar)
```
which gives the error:
```
Cannot convert `hbar` to a symengine type.
```
in `sympy.physics`